Balloon/app/device.cpp

```diff
@@ -201,8 +220,8 @@ PTCHAR CDevice::GetDevicePath(IN LPGUID InterfaceGuid)
 
     do
     {
-        cr = CM_Get_Device_Interface_List_Size(&DeviceInterfaceListLength,
-            InterfaceGuid, NULL, CM_GET_DEVICE_INTERFACE_LIST_PRESENT);
+        cr = CM_Get_Device_Interface_List_Size(
+                                 &DeviceInterfaceListLength, InterfaceGuid, NULL, CM_GET_DEVICE_INTERFACE_LIST_PRESENT);
```

Comment:
Can we not skip the line in this case? If the line too long, the parameters should be slip with the usual rule of parameter per line